src/f2pygen.py

```diff
@@ -12,7 +12,7 @@
 
 from fparser.statements import Comment
 from fparser.readfortran import FortranStringReader
-from fparser.block_statements import Select
+from fparser.block_statements import SelectCase, SelectType
```

Comment:
f2pygen now has it's own repo and is available via pip so this is a temporary fix until we remove the embedded f2pygen.py code and use the remote one. An issue explaining the problem with Select for the new version of fparser has been created in the fgenerator project so all is well.